core/new-gui/src/app/workspace/component/property-editor/property-editor.component.html

```diff
@@ -1,5 +1,61 @@
-<div class="property-editor">
-  <ndc-dynamic
-    [ndcDynamicComponent]="frameComponentConfig?.component"
-    [ndcDynamicInputs]="frameComponentConfig?.componentInputs ?? {}"></ndc-dynamic>
+<div
+  [hidden]="!isOperatorSelected"
+  cdkDrag
+  id="property-editor-container"
+  nz-resizable
+  [nzMinWidth]="400"
```

Comment:
Can you change nzMinWidth to 300 such that it will match propertyHeight.

Reply:
adjusted